projects/laji/src/app/shared-modules/select/select-subcategories/select-subcategories.component.ts

```diff
@@ -158,7 +158,9 @@ export class SelectSubcategoriesComponent implements OnChanges {
       }
       let tmpSelectedOptions = [];
       let tmpUnselectedOptions = [];
-      const countObj = this.selectedOptions[i].filter((item: SelectOptions) => item.id !== undefined).length;
+      const countObj = this.selectedOptions[i].filter(
+        (item: SelectOptions | string) => typeof item === 'string' || item.id !== undefined
+      ).length;
```

Comment:
BTW this broke the component, see task https://www.pivotaltracker.com/story/show/180599328

```typeof item === 'string' || item.id !== undefined```

is not the same as:
```item.id !== undefined```

I fixed this with:
```typeof item !== 'string' && item.id !== undefined```

Reply:
Hmm, it should be the same though. Maybe some other changes here affecting this. I pushed the above as a hotfix to prod anyways - let's have a better look next week.